apps/co2signal/co2_signal.star

```diff
@@ -0,0 +1,173 @@
+"""
+Applet: CO2 Signal
+Summary: Local power CO2 intensity
+Description: Shows the carbon intensity of your local electricity.
+Author: Harper Trow
+"""
+
+load("cache.star", "cache")
```

Comment:
Wanted to mention that we recently added cache support to the http module. Might simplify things a bit, and also future proof against potential upcoming changes to `cache.star`.